corehq/apps/users/views/mobile/users.py

```diff
@@ -743,6 +748,9 @@ def post(self, request, *args, **kwargs):
             if 'location_id' in request.GET:
                 user.set_location(loc)
 
+            if phone_number:
+                initiate_sms_verification_workflow(user, phone_number)
```

Comment:
You can't access this feature without being on a Standard plan, so that shouldn't be a problem.  I added an explicit check just in case either of those privileges change places though.